projects/ngrx.io/content/guide/store/recipes/custom-elements.md

```diff
@@ -0,0 +1,65 @@
+# Using Store with Angular Elements
+
+## Creating Angular Elements
+
+Using Store with Angular Elements is similar to using it with standard Angular Component.
+
+## Tutorial
+
+The following tutorial shows you how to manage the state of a counter, and how to select and display it within an Angular Elmements and use the same within an Angular Project and an static page. Try the <live-example name="store-elements" downloadOnly>Anguler Elements Project</live-example>.
+
+1. Create a new application by running the below command
+```sh
+ng new ngRxElementDemo --createApplication=false
+```
+2. Create a new Angular Library using below command
+```sh
+ng g library my-counter
+```
+3. Add code mentioned in [Getting Started Guide](guide/store#installation) to new created.
```

Comment:
Can you go into more detail here about what is required? Also appears to be an incomplete sentence.

Reply:
Sentence is now complete, but can you go into more depth about why they are going to the `Getting Started Guide` and what exactly they are doing?

Something like..

Follow the instructions in the `Getting Started Guide` to create a new counter store complete with `actions`, `reducer`, and `module`.